eng/pipelines/runtimelab/install-emscripten.cmd

```diff
@@ -0,0 +1,24 @@
+mkdir "%1" 2>nul
+cd "%1"
+
+git clone https://github.com/emscripten-core/emsdk.git
```

Comment:
@safern Could you please take a look at this and confirm that it is ok to do these sort of things on CI machines?